pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[project]
+name = "ecobalyse-data"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.11"
+dependencies = [
+    "bw2analyzer @ git+https://github.com/brightway-lca/brightway2-analyzer@0.11.7",
+    "bw2calc @ git+https://github.com/brightway-lca/brightway2-calc@2.0.dev17",
+    "bw2data @ git+https://github.com/brightway-lca/brightway2-data@4.0.dev42",
+    "bw2parameters @ git+https://github.com/brightway-lca/brightway2-parameters@1.1.0",
+    "bw_projects @ git+https://github.com/brightway-lca/bw_projects@v2.1.0",
+    "bw2io @ git+https://github.com/ccomb/brightway2-io@ccomb-6",
+    "pydantic-settings>=2.6.1",
+    "wrapt>=1.17.0",
+    "frozendict>=2.4.6",
+    "loguru>=0.7.2",
+    'pypardiso>=0.4.6; platform_machine!="arm64" and platform_machine!="aarch64"',
+    "jupyterlab>=4.3.2",
+    "notebook>=7.3.0",
+    "ipywidgets>=8.1.5",
+    "flatdict>=4.0.1",
+    "jupyter-collaboration>=3.0.1",
+    "typing-extensions>=4.12.2",
+    "uvicorn>=0.34.0",
+    "fastapi[all]>=0.115.6",
```

Comment:
note that `scikit-umfpack` could improve performances on macs but I couldn't figure out how to install it properly on my machine